testing/AirInterfacePowerSaver+data.json

```diff
@@ -2663,22 +2669,22 @@
                   {
                     "local-id": "000",
                     "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_MANAGEMENT",
-                    "logical-termination-point": "aips-0-0-5-op-s-bm-001"
+                    "logical-termination-point": "aips-0-0-10-op-s-bm-001"
                   },
                   {
                     "local-id": "100",
                     "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_INPUT",
-                    "logical-termination-point": "aips-0-0-5-op-s-im-000"
+                    "logical-termination-point": "aips-0-0-10-op-s-im-000"
                   },
                   {
                     "local-id": "200",
                     "port-direction": "core-model-1-4:PORT_DIRECTION_TYPE_OUTPUT",
-                    "logical-termination-point": "aips-0-0-5-op-c-bm-ro-2-0-2-001"
+                    "logical-termination-point": "aips-0-0-10-op-c-bm-ro-2-1-0.0.2"
```

Comment:
aips-0-0-10-op-c-bm-ro-2-1-0.0.2 release-number format shall be updated in all its occurences